.github/workflows/extract-translation-source-files.yml

```diff
@@ -254,9 +254,84 @@ jobs:
           # push changes to branch
           git push
 
+  # Non-Django, non-JS repositories with a generic extraction interface. This
+  # assumes that a `extract_translations` make target exists and will create
+  # a transifex_input.yaml file in the root of the site.
+  generic-translations:
+    strategy:
+      # using max-parallel to avoid git push/pull issues when running in parallel
+      max-parallel: 1
+      matrix:
+        repository_config:
+          - repo: tutor-contrib-aspects
+            transifex_file_path: transifex_input.yaml
```

Comment:
That looks good. 

I'm not sure if it's widely documented, but that's a proper GitHub matrix config to set the item as an object instead of a string.